docs/reference/commandline/plugin_push.md

```diff
@@ -38,7 +38,7 @@ user/plugin             latest              A sample plugin for Docker   false
 $ docker plugin push user/plugin
 ```
 
-## Related information
+# Related information
```

Comment:
the documents in `docs/` should not be changed, because the new documentation rendering engine generates the H1 from the metadata at the top (in this case, the H1 is generated from `title: "plugin push"`